arm-monitor/2016-03-01/swagger/serviceDiagnosticsSettings_API.json

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
It adds a delete operation, it changes the signature of the get and put calls (there is one more parameter in those calls, the name.)
Certainly this could be done with an optional parameter with a default value of "service".
Vinicius also change the name of the calls, he removed the "Service" prefix. I could revert that part of the change and users of the SDK won't see differences in their calls.

Please let me know of that is a better way to do it.